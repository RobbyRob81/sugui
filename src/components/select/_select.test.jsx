import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Select from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<Select items={[{ val: 1, label: 'one' }, { val: 2, label: 'two', selected: true }, { val: 3, label: 'three' }]} />);
  expect(container.firstChild).toMatchSnapshot();
});
