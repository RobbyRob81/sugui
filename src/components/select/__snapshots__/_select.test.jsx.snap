// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  class="sc-iwsKbI fzPyAB"
>
  <div>
     
    <div
      class="sc-EHOje iTZenT"
    >
      <input
        class="sc-bZQynM gDOsNr"
        readonly=""
        value="two"
      />
    </div>
  </div>
  <svg
    class="sc-jTzLTM gtMgLG"
    height="31"
    viewBox="0 0 32 31"
    width="32"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fill-rule="evenodd"
    >
      <path
        d="M0 0h32v30.974H0z"
        fill="none"
        stroke="none"
      />
      <path
        d="M27.333 10.97L16 21.94 4.667 10.97"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </g>
  </svg>
  <section
    class="sc-gZMcBi dauSsJ sc-dnqmqq fRcCcF"
  >
    <ul
      class="sc-gqjmRU RgYKB"
    >
      <li
        class="sc-VigVT cjldzT"
      >
        one
      </li>
      <li
        class="sc-VigVT cjldzT"
      >
        two
      </li>
      <li
        class="sc-VigVT cjldzT"
      >
        three
      </li>
    </ul>
  </section>
</div>
`;
